packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -246,4 +246,8 @@ contract TaikoL1 is
     function getVerifierName(uint16 id) public pure returns (bytes32) {
         return LibUtils.getVerifierName(id);
     }
+
+    function getOracleProver() public view returns (address) {
```

Comment:
If we add this function, shall we also add more for other named addresses?

Reply:
I don't necessarily need them, but we could actually just do a generic function.
Ie:

function resolveAddressForName(string name) returns (address)

And then we can just pass in "oracle_prover", etc